### ISSUE TITLE
What is difference between "fine-tuning" and "intermediate fine-tuning" ?

### ISSUE THREAD

**Issue by adaxigua** [dvlab-research/MOOD#5]
Q1 ：Are they right?

[1] intermediate fine-tuning is working on  ImageNet-21K?

[2] fine-tuning is working on one-class ID-dataset?

Q2：where are they related code?

the paper say:

> 1. Pre-train the Masked Image Modeling ViT on ImageNet-21k
> 2. Apply intermediate fine-tuning ViT on ImageNet-21k.
> 3. Apply fine-tuning of pre-trained ViT on the ID dataset.

the code contains:

>a. Pretrained models
>b. Fine-tuning on In-Distribution Dataset

In my view, 'a' is for '1' and 'b' is for '3', where is the code for '2' ?
